ci/continuous-builder.sh

```diff
@@ -15,12 +15,9 @@ if [ "$1" != --skip-setup ]; then
     fi
   done
 
-  case "$(uname)" in
-    # The MacOS builders aren't in the right directory at first, apparently.
-    Darwin) ci_work_dir=/build/ci_checks/alice_ci_$WORKER_INDEX
-            mkdir -p "$ci_work_dir"
-            cd "$ci_work_dir" || exit 1;;
-  esac
+  if [ -r ~/.continuous-builder.sh ]; then
```

Comment:
I would simply drop it. My preferred solution is that we either run aurora on mac as well (we might have to update our own mesos quite a bit though) or that we use launchd.

Reply:
Right. For now, that file apparently contains secrets that we need. As above, is it OK if we keep the old way for now so the builds work again?

Reply:
agreed.